README.md

```diff
@@ -1,2 +1,37 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+
+![It's a FLOWCHART! Beware the ED! :D](https://github.com/TMGilchrist/comp110-worksheet-B/blob/master/Fallout%20minigame%20flowchart.png)
```

Comment:
Flowchart looks good. I would maybe put the initialisation of `n` closer to where it is used, so that it is clearer what its purpose is (particularly since its name does not give any clues).